python/databricks/bundles/apps/__init__.py

```diff
@@ -0,0 +1,209 @@
+    "AppResourceGenieSpaceParam",
+    "AppResourceJob",
+    "AppResourceJobDict",
+    "AppResourceJobJobPermission",
+    "AppResourceJobJobPermissionParam",
+    "AppResourceJobParam",
+    "AppResourceParam",
+    "AppResourceSecret",
+    "AppResourceSecretDict",
+    "AppResourceSecretParam",
+    "AppResourceSecretSecretPermission",
+    "AppResourceSecretSecretPermissionParam",
+    "AppResourceServingEndpoint",
+    "AppResourceServingEndpointDict",
+    "AppResourceServingEndpointParam",
+    "AppResourceServingEndpointServingEndpointPermission",
+    "AppResourceServingEndpointServingEndpointPermissionParam",
+    "AppResourceSqlWarehouse",
+    "AppResourceSqlWarehouseDict",
+    "AppResourceSqlWarehouseParam",
+    "AppResourceSqlWarehouseSqlWarehousePermission",
+    "AppResourceSqlWarehouseSqlWarehousePermissionParam",
+    "AppResourceUcSecurable",
+    "AppResourceUcSecurableDict",
+    "AppResourceUcSecurableParam",
+    "AppResourceUcSecurableUcSecurablePermission",
+    "AppResourceUcSecurableUcSecurablePermissionParam",
+    "AppResourceUcSecurableUcSecurableType",
+    "AppResourceUcSecurableUcSecurableTypeParam",
+    "ApplicationState",
+    "ApplicationStateParam",
+    "ApplicationStatus",
+    "ApplicationStatusDict",
+    "ApplicationStatusParam",
+    "ComputeSize",
+    "ComputeSizeParam",
+    "ComputeState",
+    "ComputeStateParam",
+    "ComputeStatus",
+    "ComputeStatusDict",
+    "ComputeStatusParam",
+    "Lifecycle",
+    "LifecycleDict",
+    "LifecycleParam",
+]
+
+
+from databricks.bundles.apps._models.app import App, AppDict, AppParam
+from databricks.bundles.apps._models.app_deployment import (
+    AppDeployment,
+    AppDeploymentDict,
+    AppDeploymentParam,
+)
+from databricks.bundles.apps._models.app_deployment_artifacts import (
+    AppDeploymentArtifacts,
+    AppDeploymentArtifactsDict,
+    AppDeploymentArtifactsParam,
+)
+from databricks.bundles.apps._models.app_deployment_mode import (
+    AppDeploymentMode,
+    AppDeploymentModeParam,
+)
+from databricks.bundles.apps._models.app_deployment_state import (
+    AppDeploymentState,
+    AppDeploymentStateParam,
+)
+from databricks.bundles.apps._models.app_deployment_status import (
+    AppDeploymentStatus,
+    AppDeploymentStatusDict,
+    AppDeploymentStatusParam,
+)
+from databricks.bundles.apps._models.app_permission import (
+    AppPermission,
+    AppPermissionDict,
+    AppPermissionParam,
+)
+from databricks.bundles.apps._models.app_permission_level import (
+    AppPermissionLevel,
+    AppPermissionLevelParam,
+)
+from databricks.bundles.apps._models.app_resource import (
+    AppResource,
+    AppResourceDict,
+    AppResourceParam,
+)
+from databricks.bundles.apps._models.app_resource_database import (
+    AppResourceDatabase,
+    AppResourceDatabaseDict,
+    AppResourceDatabaseParam,
+)
+from databricks.bundles.apps._models.app_resource_database_database_permission import (
```

Comment:
there are a bunch of fields with repetition `app_resource_database_database_permission`, `app_resource_genie_space_genie_space_permission`, ...